Samples/HttpTrigger1/function.json

```diff
@@ -3,7 +3,7 @@
   "disabled": false,
   "bindings": [
     {
-      "authLevel": "function",
+      "authLevel": "anonymous",

```

Comment:
Nit: Change  authlevel to function as that's the default

Reply:
Sure, will do